src/UI/UIView.cpp

```diff
@@ -615,7 +658,7 @@ void UIView::draw_ui_callback()
         std::string filePath = fileDialogLoadTrnsFnc.selected_path;
         std::ifstream fileToLoad(filePath);
         load_trans_functions(fileToLoad);
-        m_table_selection = 0;
+        m_trnfnc_table_selection = 0;
```

Comment:
This line is showing up in multiple `if` blocks here. Is there a way to change/simplify the logic so you only have to call `m_trnfnc_table_selection = 0;` one time?